sphinx_doc/howtos/tutorials/docs/navigation2_with_turtlebot3_in_gazebo.rst

```diff
@@ -0,0 +1,193 @@
+.. _navigation2-with-turtlebot3-in-gazebo:
+
+Navigation2 with Turtlebot 3 in Gazebo
+**************************************
+
+- `Overview`_
+- `Requirements`_
+- `Tutorial Steps`_
+- `Videos`_
+
+Overview
+========
+
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2 in Gazebo.
+Turtlebot is a low-cost, personal robot kit with open-source software (ROS2).
+Turtblebot robots are widely supported by the ROS community.
+You can find many applications and examples of Turtlebot projects on the Internet.
+You can find more information about Turtlebot3 `here. <https://www.turtlebot.com/>`_
+
+Links to the robot kits and Github repo.
+
+- `Turtlebot Burger <http://www.robotis.us/turtlebot-3-burger-us/>`_
+- `Turtlebot Waffle <http://www.robotis.us/turtlebot-3-waffle-pi/>`_
+- `Turtlebot 3 Github <https://github.com/ROBOTIS-GIT/turtlebot3>`_
+
+This tutorial consists of two parts.
+In the first part, you will learn how to use Turtlebot 3 robots in simulation (Gazebo).
+And in the second part, you will learn how to control a real Turtlebot Waffle using Navigation2.
+
+``ROS2 Dashing`` and ``Navigation2 Dashing 0.2.4`` are used to create this tutorial.
+
+This tutorial may take about 30 minutes to complete. It depends on your experience with ROS, robots, and what computer system you have.
```

Comment:
change "on your" to It depends "on the user" experience with ..

Reply:
IMO, the second-person perspective puts the reader into the story and makes this tutorial more readable especially for beginners. I am trying to use empathy to connect with the reader telling them "hey I know you are a beginner and don't know much about this tool, but we will get this done step by step together".